css/fonts.css

```diff
@@ -0,0 +1,37 @@
+@font-face {
+  font-family: 
+  "Montserrat";
+  src: 
+  url("../fonts/Montserrat-Bold.woff2") format("woff2"),
+  url("../fonts/Montserrat-Bold.woff") format("woff");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5705021)